### ISSUE TITLE
Problem with Apple TV app only when used with Linux server

### ISSUE THREAD

**Issue by sfatula** [ValveSoftware/steam-for-linux#6773]
#### Your system information

* Steam client version (build number or date): Apple TV app 1.1.44
* Distribution (e.g. Ubuntu): Apple TVOS
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Opened ticket HT-8BV5-5PDG-P3RB with Steam, but they sent me here. 

So, I have a Linux steam server. Nvidia RTX2060. I have an Apple TV (non 4k) client running Steam Link. That is connected to 5G, and steam tests the connection as 100Mbps. I also have an IOS client, just for comparison, and it has the same issue I shall describe.

When I am on the very first screen after connecting with the Apple TV app (or IOS), where you can select settings, library, etc., the screen flickers or is choppy in big picture mode. Regularly, always. So, you might blame the connection. However, note that when I go into a game, where it actually uses much more resources and bandwidth, then it's perfect. I suspect this is some sort of Steam Link app bug, or a server app bug. The bandwidth being used when showing the home screen or library is very low. I have tried all sorts of setting adjustments to no avail.

But also please note, this does not happen when connecting the same Apple TV (or IOS) to a Windows server. So, it also could be some sort of issue with Linux streaming only I suppose.

So, to recap, it is not while playing a game, it is when browsing the library or on the big picture home screen.

#### Steps for reproducing this issue:

1. Open Steam Link app on Apple TV
2. Select a linux server
3. Connects, and, screen flickers every so often

**Comment by kisak-valve** [comment]
Hello @sfatula, this reads like #4351. Please test if unchecking "Allow Flipping" in nVidia's settings has an effect.

**Comment by sfatula** [comment]
Yeah, you're right. I poked around but apparently used the wrong words for my searches.
